SU2_CFD/src/solvers/CSolver.cpp

```diff
@@ -4209,3 +4215,109 @@ void CSolver::BasicLoadRestart(CGeometry *geometry, const CConfig *config, const
                    string("It could be empty lines at the end of the file."), CURRENT_FUNCTION);
   }
 }
+
+#ifdef HAVE_LIBROM
+void CSolver::SavelibROM(CSolver** solver, CGeometry *geometry, CConfig *config, bool converged) {
+  
+  unsigned long iPoint, total_index;
+  unsigned short iVar;
+  
+  bool unsteady            = (config->GetTime_Marching() != TIME_MARCHING::STEADY);
+  string filename          = config->GetlibROMbase_FileName();
+  unsigned short pod_basis = config->GetKind_PODBasis();
+  unsigned long TimeIter   = config->GetTimeIter();
+  unsigned long nTimeIter  = config->GetnTime_Iter();
+  int maxBasisDim          = config->GetMax_BasisDim();
+  int dim = int(nPointDomain * nVar);
```

Comment:
Better check for unsteady with GetTime_Domain() (or something similar, can't recall the exact name).

const, or const auto as appropriate, the things you get from the config.